plugins/basic/python/tk_photoshopcc_basic/__init__.py

```diff
@@ -8,4 +8,5 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-from .plugin_bootstrap import bootstrap_toolkit
\ No newline at end of file
+from .classic_init import toolkit_classic_bootstrap
+from .plugin_init import toolkit_plugin_bootstrap
```

Comment:
Also, is there a need for the `tk_photoshopcc_basic` module level in the `python` directory?